### DISCUSSION TITLE
Verschieben der Produktbilder auf Produktseiten in der mobilen Ansicht über den Preisvergleich

### DISCUSSION THREAD

**Discussion by srzjnw** [affiliatetheme/affiliatetheme#12] [Ideas]
In der aktuellen Version des Affiliatetheme werden in der Desktop- und Tabletansicht die Produktbilder auf Produktseiten prominent oben links angezeigt, sodass der Besucher sofort erkennen kann, dass er auf der richtigen Seite ist.

In der mobilen Ansicht werden jedoch der Preisvergleich und die Produkteigenschaften über das Produktbild geschoben, wodurch die visuelle Bestätigung für den Besucher fehlt.

Ich würde mir wünschen, dass die Produktseite in der mobilen Ansicht so angepasst wird, dass die Produktbilder über dem Preisvergleich angezeigt werden. Auf diese Weise erhält man eine visuelle Bestätigung und einen Call-to-Action "Above-the-fold".

Dies würde voraussichtlich zu einer geringeren Absprungrate auf Produktseiten für mobile Besucher führen, also möglicherweise bei mehr als der Hälfte aller Aufrufe.

![1desktop - Kopie](https://user-images.githubusercontent.com/5694842/223483990-16220afe-de6a-47e2-92a5-6513d938e1a3.png)

![2mobile - Kopie](https://user-images.githubusercontent.com/5694842/223484800-4a9c9886-d746-4486-b5a5-13f8522b4977.png)

**Comment by endcoreAK** [comment]
Bis wir die Umsetzung angehen, könntest du das ganze mit einem kleinen CSS Snippet lösen:

```
@media (max-width: 767px) {
    #product-details { display: flex; flex-direction: column; flex-wrap: wrap; }
    #product-details .col-sm-push-4 { order: 2; margin-top: 2rem; }
    #product-details .col-sm-push-8 { order: 1; }
}
```

... und damit dann der Kaufen-Button auch noch im sichtbaren Bereich ist, könnte man die Thumnails der Gallery auf eine Reihe begrenzen und scrollbar machen:

```
@media (max-width: 767px) {
    #productGallery .carousel-indicators { display: flex; flex-direction: row; flex-wrap: nowrap; overflow: scroll; max-width: 90vw; }
}
```

<img width="325" alt="Bildschirm­foto 2023-03-08 um 08 15 08" src="https://user-images.githubusercontent.com/6432237/223646539-41ce5fd4-c7cb-40b9-a963-072c17a1a168.png">

@StrongLucky bitte mal als Option für's Theme im Hinterkopf behalten. :)

**Comment by srzjnw** [comment]
Top! Funktioniert.
Wenn ich es mir aussuchen dürfte, hätte ich den Titel gerne über den Produktbildern, aber das sind nur noch Details. Danke.

**Comment by endcoreAK** [comment]
@sovus  das geht per rein CSS leider nicht, weil wir quasi 2 Container haben, einmal die Gallery und einmal die Info (inkl. Headline, Rating und Buybox). Behalten wir aber im Hinterkopf. :)
